pkg/common/ut/request_test.go

```diff
@@ -116,3 +117,13 @@ func createChunkedBody(body []byte) []byte {
 	}
 	return append(b, []byte("0\r\n\r\n")...)
 }
+
+func TestPerformBizServerRequest(t *testing.T) {
+	h := server.Default()
+	h.GET("/ping", Ping)
+	w := PerformBizServerRequest(h, "GET", "/ping", &Body{bytes.NewBufferString("1"), 1},
```

Comment:
`w := PerformRequest(h.Engine, "GET", "/ping", &Body{bytes.NewBufferString("1"), 1},` is equal to line 124?